src/guide/lnurl_pay_service.md

```diff
@@ -0,0 +1,91 @@
+# Receiving payments using LNURL-Pay
+
+Breez SDK - *Liquid* users have the ability to receive Lightning payments using [LNURL-Pay](https://github.com/lnurl/luds/blob/luds/06.md).
+
+LNURL-Pay requires a web service that serves LNURL-Pay requests. This service needs to communicate with the SDK in order to fetch the necessary metadata data and the associated payment request.
+To interact with the SDK, the service uses a simple protocol over push notifications:
+* The service sends a push notification to the user's mobile app with the LNURL-Pay request and a reply URL.
+* The app responds to reply URL with the required data.
+* The data is forwarded from the service to the payer.
```

Comment:
To be more inline with other items; 
- **service** _sends_
- **app** _responds_
```suggestion
* The service forwards the data to the payer.
```